app/upgrades/v0.0.13.go

```diff
@@ -0,0 +1,74 @@
+package upgrades
+
+import (
+	"context"
+
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	cosmosTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/pokt-network/poktroll/app/keepers"
+	migrationtypes "github.com/pokt-network/poktroll/x/migration/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+const Upgrade_0_0_13_PlanName = "v0.0.13"
+
+// Upgrade_0_0_13 handles the upgrade to release `v0.0.13`.
+// This is planned to be issued on both Pocket Network's Shannon Alpha & Beta TestNets.
+var Upgrade_0_0_13 = Upgrade{
+	PlanName: Upgrade_0_0_13_PlanName,
+	CreateUpgradeHandler: func(mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+		applyNewParameters := func(ctx context.Context) (err error) {
+			logger := cosmosTypes.UnwrapSDKContext(ctx).Logger()
+			logger.Info("Starting parameter updates", "upgrade_plan_name", Upgrade_0_0_13_PlanName)
+
+			sharedParams := keepers.SharedKeeper.GetParams(ctx)
+			sharedParams.GatewayUnbondingPeriodSessions = sharedtypes.DefaultGatewayUnbondingPeriodSessions
+			err = keepers.SharedKeeper.SetParams(ctx, sharedParams)
```

Comment:
@red-0ne you've previously suggested doing a `ValidateBasic` - let's start with this upgrade! :)